src/index.ts

```diff
@@ -63,15 +78,16 @@ class LinkSDK {
 
   // Open My Account to authorize application to use MtLink API
   authorize(options: MyaccountOptions = {}): void {
-    const { newTab = false, email, authPage } = options;
+    const { newTab = false, backTo, email, authPage } = options;
     const { PATHS: { OAUTH }} = MY_ACCOUNT;
     const configs = {
       sdk_platform: 'js',
       email,
```

Comment:
email itself have to be encoded first because email spec can have weird characters in it too including all valid query string delimiters

Reply:
Should be encoded within `endoeConfig` with `encodeURIComponent` isn't it?

Reply:
I can confirm it gets encoded

Reply:
it looks like `qs.stringify` already encoding this email properly, so we might need to fix something on my account too, if we do not encode the email before